resources/assets/pages/ShowAction.js

```diff
@@ -0,0 +1,49 @@
+import gql from 'graphql-tag';
+import React, { useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { useQuery } from '@apollo/react-hooks';
+
+import Action from '../components/Action';
+import Shell from '../components/utilities/Shell';
+import MetaInformation from '../components/utilities/MetaInformation';
+
+const SHOW_ACTION_QUERY = gql`
+  query ShowActionQuery($id: Int!) {
+    action(id: $id) {
+      id
+      name
+      campaignId
```

Comment:
Should we include other fields (like `noun`, `verb`, `reportback`, etc.) in this query? Although I guess we'd have to add some conditionals for camel-cased vs. snake-cased formatting, since the other place this `Action` component is used is still using the plain REST API...

Reply:
Yes! This has been resolved in #960 by rendering Actions via GraphQL from both this permalink and the campaign permalink.